pkg/common/providers/ocmprovider/cluster.go

```diff
@@ -360,16 +361,17 @@ func (o *OCMProvider) DetermineMachineType(cloudProvider string) (string, error)
 
 		for range machinetypes.Items().Slice() {
 			machinetypeObj := machinetypes.Items().Slice()[rand.Intn(machinetypes.Total())]
-			computeMachineType = machinetypeObj.ID()
+			returnedType = machinetypeObj.ID()
 			break
 		}
+	}
```

Comment:
So if we don't take the `if` statement above, when we get here we set the machine type to the empty string. Is that the intended behavior? Does empty string have semantics like "use the default" that I don't know about?